sky/shell/platform/android/vk/vulkan/vk_platform.h

```diff
@@ -0,0 +1,120 @@
+//
+// File: vk_platform.h
+//
+/*
+** Copyright (c) 2014-2015 The Khronos Group Inc.
+**
+** Licensed under the Apache License, Version 2.0 (the "License");
+** you may not use this file except in compliance with the License.
+** You may obtain a copy of the License at
+**
+**     http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
These files should go in third_party in the buildroot.

Do these files exist in an git repo somewhere we can DEPS in?  It seems strange that we'd need to have our own copy of these system headers.